frontend/src/lib/i18n/en.json

```
@@ -361,6 +361,12 @@
     "create_as_public_tooltip": "Public neurons reveal more information about themselves including how they vote on proposals.",
     "create_as_public_neuron_failure": "Making your neuron public has failed, so it was created as a private neuron. You can change neuron visibility at any time under \"Advanced Details & Settings\""
   },
+  "losing_rewards_banner": {
+    "days_left_title": "$left left to confirm your neuron following",
```

[nitpick]: the first time I read the component, I thought `left` referred to location rather than time remaining.
```suggestion
    "days_left_title": "$timeLeft left to confirm your neuron following",
```

---

Good point, renamed.